llvm/include/llvm/ADT/StableHashing.h

```diff
@@ -8,93 +8,47 @@
 //
 // This file provides types and functions for computing and combining stable
 // hashes. Stable hashes can be useful for hashing across different modules,
-// processes, or compiler runs.
+// processes, or compiler runs for a specific version. It currently employs
+// the xxh3_64bits hashing algorithm. Be aware that this implementation may be
+// adjusted or updated as improvements to the system are made.
```

Comment:
Should we also add that these are stable across machines? My other edits are to clarify we mean the hash could change if the compiler changes. Let me know if these changes make sense!

```suggestion
// processes, machines, or compiler runs for a specific compiler version. It currently employs
// the xxh3_64bits hashing algorithm. Be aware that this implementation may be
// adjusted or updated as improvements to the compiler are made.
```